docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

```diff
@@ -64,17 +64,17 @@ Azure Service Fabric
 
 ![https://azure.microsoft.com/svghandler/service-fabric?width=600&height=315](./media/image26.png)
 
-> [Service Fabric](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-overview/) is a Microsoft microservices platform for building applications. It is an [orchestrator](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-cluster-resource-manager-introduction/) of services and creates clusters of machines. By default, Service Fabric deploys and activates services as processes, but Service Fabric can deploy services in Docker container images. More importantly, you can mix services in processes with services in containers in the same application.
+> [Service Fabric](https://azure.microsoft.com/documentation/articles/service-fabric-overview/) is a Microsoft microservices platform for building applications. It is an [orchestrator](https://azure.microsoft.com/documentation/articles/service-fabric-cluster-resource-manager-introduction/) of services and creates clusters of machines. By default, Service Fabric deploys and activates services as processes, but Service Fabric can deploy services in Docker container images. More importantly, you can mix services in processes with services in containers in the same application.
 >
 > As of May 2017, the feature of Service Fabric that supports deploying services as Docker containers is in preview state.
 >
-> Service Fabric services can be developed in many ways, from using the [Service Fabric programming models ](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-choose-framework/)to deploying [guest executables](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-deploy-existing-app/) as well as containers. Service Fabric supports prescriptive application models like [stateful services](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-reliable-services-introduction/) and [Reliable Actors](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-reliable-actors-introduction/).
+> Service Fabric services can be developed in many ways, from using the [Service Fabric programming models ](https://azure.microsoft.com/documentation/articles/service-fabric-choose-framework/)to deploying [guest executables](https://azure.microsoft.com/documentation/articles/service-fabric-deploy-existing-app/) as well as containers. Service Fabric supports prescriptive application models like [stateful services](https://azure.microsoft.com/documentation/articles/service-fabric-reliable-services-introduction/) and [Reliable Actors](https://azure.microsoft.com/documentation/articles/service-fabric-reliable-actors-introduction/).
```

Comment:
review the remainder https://azure.microsoft.com/documentation/ links to go to the new docs.microsoft.com link without en-us